### DISCUSSION TITLE
v0.6.0

### DISCUSSION THREAD

**Discussion by SoCuul** [SoCuul/SCInsta#93] [Announcements]
> Tested on Instagram v350.0.12
> Media downloading features are currently being rewritten! Expect to see them fully working in a future release.

## New Features:
- Disable scrolling reels (can view a singular one, but can't scroll to view more) 44ade451f1daeffcd9e97434a4ade8f38a463214
- Hide the entire home feed (including posts) 2bb34a10caba12e81bb5fef1f2da914017b497a6

## Fixes:
- Fix hide meta ai functionality in newest Instagram version 066c63f5d8e8f6bed96609d829173fa5c18903e6
- Improve hide story tray functionality c5369b9ca71382ed76705c55bad258a296bf4e6c

---

**Full Changelog**: https://github.com/SoCuul/SCInsta/compare/v0.5.1...v0.6.0

<hr /><em>This discussion was created from the release <a href='https://github.com/SoCuul/SCInsta/releases/tag/v0.6.0'>v0.6.0</a>.</em>

**Comment by CoffiPants** [comment]


**Reply by CoffiPants** [reply]
when I upload ipa with sideloadly it shows me this

**Reply by peppahacker** [reply]
What is your problem? I cannot to see the image

**Comment by peppahacker** [comment]
Amazing tweak, looks clean
![IMG_3522](https://github.com/user-attachments/assets/b9dc80e9-3a22-4456-9d91-8839cc5bd52f)
![IMG_3521](https://github.com/user-attachments/assets/7363bf29-c1c0-4705-a449-2d2d5e24882c)


I will donate 100% for your amazing work, guys! This will save me a lot of time instead of sitting on the toilet watching stupid reels! I haven't looked under the hood, but I hope you're not fetching login information, etc. 

Again, keep doing what you're doing!

**Reply by SoCuul** [reply]
Thank you so much! ❤️
I try my best haha, though I'm still not nearly as experienced with tweak development as I would like to be.

Also, your point about the login information is the exact reason I forked the original BHInstagram project. I find it hard to trust modified apps from unknown individuals, with no guarantee of it being safe. Stuff like this should be open-source, for visibility reasons, especially when you're entering account information which you need to keep secure.

**Reply by furua7** [reply]
How were you able to change the settings? I can't the place for it anywhere

**Comment by sayyted** [comment]
I need help on how to get it

**Reply by SoCuul** [reply]
The easiest way is to grab a version of Instagram version 350, and then follow the steps outlined in [this guide](https://github.com/SoCuul/SCInsta/wiki/Building-IPA) to build the tweak into an installable IPA of SCInsta.

**Comment by AZ-q** [comment]
hello i need help with installing SCInsta with IPA i did every steps to create IPA file but i dont know what should i do after the workflow shows a green check 
![image](https://github.com/user-attachments/assets/d5257d98-b06c-4a0e-94a9-9ffe2160a484)

**Reply by Matsuui** [reply]
If you click on "Build and Package SCInsta" there should be a box that says "Artifacts" with a download button. It will download a .zip that will have the .ipa file in it.

**Reply by AZ-q** [reply]
yeah it works thx but why the keep deleted message isnt work can i update it by myself if i can help me please and tell me how

**Reply by SoCuul** [reply]
Keep deleted messages is working for me just the same as it always has, so I'm not sure why it wouldn't work on your end.

**Comment by furua7** [comment]
How do I change the settings? I can't find the SCinsta settings anywhere

**Reply by SoCuul** [reply]
Do you see the Instagram text logo on the home screen of the app?

**Reply by harounai** [reply]
Same here, it works on v380.0 but the long press on the 'instagram' logo on the homepage doesn't.
